gril/grilreply.c

```diff
@@ -1456,3 +1461,99 @@ struct parcel_str_array *g_ril_reply_oem_hook_strings(GRil *gril,
 out:
 	return str_arr;
 }
+
+struct reply_radio_capability *g_ril_reply_parse_get_radio_capability(
```

Comment:
Considering the fact that upstream nak'd this whole approach, perhaps it would be better to not continue to add new functions to grilreply/request/unsol, and move this code inline where it's used?

Also, no unit tests exist for this ( or for any of the other message changes ).